app/controllers/users_controller.rb

```diff
@@ -1,6 +1,13 @@
 class UsersController < ApplicationController
+  rescue_from JsonApiClient::Errors::ClientError, with: :render_providers
+  rescue_from JsonApiClient::Errors::ServerError, with: :render_providers
+
   def accept_transition_info
     User.member(current_user['user_id']).accept_transition_screen
+    render_providers
+  end
+
+  def render_providers
```

Comment:
I like that we're not exposing errors to end-users, but it'd be good to send the errors to Sentry in this method, so we don't eat them